features/dd-sdk-android-trace/consumer-rules.pro

```diff
@@ -1,3 +1,4 @@
 -keepnames class io.opentracing.util.GlobalTracer {
     public boolean isRegistered();
 }
+-keepclassmembernames class org.jctools.** { *; }
```

Comment:
jctools classes are at the core of the CoreTracer consumer and we cannot easily get rid of. On the other hand jctools is not really compatible with Proguard as it is using Reflection under the hood and we will need to keep at least the class member names.